tests/conftest.py

```diff
@@ -29,7 +32,7 @@ def db():
     try:
         cursor.execute('CREATE DATABASE ocdskingfishercolab_test')
```

Comment:
Do we only need `psycopg2` to run this test scaffolding? Or can ipython-sql not do it?

Reply:
Yes, its only used here.

I kept this as is, mostly because it was easiest. Particularly, avoiding the [transaction block issues](https://github.com/open-contracting/kingfisher-colab/pull/36/files/cfd85cbbbbae3e06116b591c45e96daeb2053684#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128L26-R30) with ipython-sql looked to be fiddly, and [proper support for closing connections is broken](https://github.com/catherinedevlin/ipython-sql/issues/169).

Reply:
The first link doesn't seem to highlight lines anymore, but sounds good to leave as-is.